background_scripts/completion_search.coffee

```diff
@@ -42,7 +42,7 @@ CompletionSearch =
 
   # The amount of time to wait for new requests before launching the current request (for example, if the user
   # is still typing).
-  delay: 100
+  delay: 0
```

Comment:
So that's the 50ms not being long enough for our processing sometimes. Processor speed and OS load seems like a bad way to decide throttling. Lets change the delay in the frontend to reflect what we think reasonable throttling is when we're using completers.